protos/flyteidl/core/security.proto

```diff
@@ -21,20 +21,25 @@ message Secret {
         FILE = 2;
     }
 
-    // The name of the secret to mount. This has to match an existing secret in the system. It's up to the implementation
-    // of the secret management system to require case sensitivity.
+    // The name of the secret group where to find the key referenced above.
```

Comment:
the name of the secret?
And write examples maybe
K8s secrets, the name of the secret
in confidant - Credential
In Secret manager: name of the secret